.github/workflows/validate-pr-title.yaml

```diff
@@ -32,8 +32,4 @@ jobs:
             refactor
             revert
             test
-          scopes: |
-            harness
-            testing
-            tracing
-          requireScope: false
+          scopes: ""
```

Comment:
This works. I've added a scope "(foo)" and CI fails: https://github.com/canonical/operator/actions/runs/15196060716/job/42740323103?pr=1748

```
Error: Unknown scope "foo" found in pull request title
```